lhctl/cmd/principal.go

```diff
@@ -148,6 +196,12 @@ func init() {
 	putPrincipalCmd.Flags().Bool("overwrite", false, "Overwrites principal information")
 	putPrincipalCmd.Flags().String("tenantId", "", "Tenant associated with the principal")
 
+	searchCmd.AddCommand(searchPrincipalCmd)
+	searchPrincipalCmd.Flags().String("tenantId", "", "List Principals associated with this Tenant ID")
+	searchPrincipalCmd.Flags().Bool("isAdmin", false, "List only Principals that are admins")
+	searchPrincipalCmd.Flags().Int("earliestMinutesAgo", -1, "Search only for Principals that were created no more than this number of minutes ago")
+	searchPrincipalCmd.Flags().Int("latestMinutesAgo", -1, "Search only for Principals that were created at least this number of minutes ago")
+
```

Comment:
```suggestion
searchPrincipalCmd.MarkFlagsOneRequired("isAdmin", "tenantId")
```

Reply:
I believe this can replace line 91.
see [Command.MarkFlagsOneRequired](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsOneRequired)